content/en/network_performance_monitoring/guide/manage_traffic_costs_with_npm.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Manage Traffic Costs with NPM
+kind: guide
+---
+Traffic is expensive, especially in the cloud. Cloud providers charge differently for traffic within an availability zone, traffic between availability zones (AZs), and traffic between regions (this pricing may differ by region). Cross-regional traffic is not only the most expensive, but also the most vulnerable to errors, latency, and security threats. 
+
+Network Performance Monitoring allows you to track all of the traffic patterns described above by mapping dependencies between any tags in Datadog, including service, container, availability zone, region, datacenter, etc… This insight into your dependencies and the amount of traffic volume they produce (which is ultimately what you are billed on) can be used to monitor and optimize on your traffic-related costs. 
+
+## Datadog’s story
+
+When we began our Kubernetes migration, migrating stateless services was (expectedly) much faster and easier than migrating our stateful services (e.g. Kafka), so we migrated our stateless services first. The outcome was terabytes of cross-AZ traffic between the stateful services (all in one AZ) and stateless services (spread across the other AZs), but then our cloud bill exploded. We used our own Network Product to identify the root cause - a suboptimal migration strategy - to optimize our network (by sharding our stateful services) and ultimately saw significant reductions in our cloud provider costs.
+
+## Steps for managing traffic costs 
+
+1. To find similar issues in your own environment, you can start by grouping your traffic flows by regions, 
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/region.png" alt="Group flows by region">}}
```

Comment:
one screenshot is enough here to convey the point - let's keep region and remove the rest